text/0308-cli-advisories.md

```diff
@@ -0,0 +1,280 @@
+    "component": ["framework"],
+    "version": "<1.126.0"
+  }
+```
+
+3. Create a PR with this change and wait for an approval. Only SDMs have
+   permission to approve PRs in this repository.
+4. When the PR gets merged, the advisory will be visible to all CLI
+   installations. The GitHub issue will also be automatically updated with the
+   information contained in the file. All the necessary tags will also be added
+   automatically. 
+5. You can keep updating the issue normally, as new information comes in, but
+   you're not allowed to touch the sections auto-generated from the advisories
+   file.
+
+[ ] Signed-off by API Bar Raiser @xxxxx
+
+## Public FAQ
+
+### What are we launching today?
+
+A new communication channel between AWS and users of the CDK. Starting on
+version x.y.z of the CDK CLI, customers will be notified, on every command,
+about security vulnerabilities, regressions and usage of unsupported versions.
+
+### Why should I use this feature?
+
+These advisories shown by the CLI contain very important and actionable
+information about problems that directly affect you. They give you an
+opportunity to upgrade the CLI or the construct library when necessary or to
+work around high-impacting issues.
+
+## Internal FAQ
+
+### Why are we doing this?
+
+In case of emergency announcements, such as security vulnerabilities or
+regressions, the only mechanisms we have right now are email campaigns and
+pinned GitHub issues. These are not always the best means to reach out to
+customers. Many email addresses are not monitored by anyone and customers don’t
+necessarily check the GitHub repository for updates. The output of the CLI, on
+the other hand, is seen by many customers every day.
+
+### Why should we *not* do this?
+
+This is a powerful feature to convey urgent and important messages to customers.
+However, to keep its relevance, it should be used sparingly and the messages
+must meet a high bar. Otherwise it will just become a tool for spamming
+customers. We will introduce filters to reduce this risk, by only showing
+content that applies to each particular environment and also require PR approval 
+for any changes in advisories. But ultimately, it hinges on responsible use. In 
+particular, this feature should not be used for things like marketing campaigns, 
+blog post announcements and things like that. If the mechanisms proposed in this
+RFC are not considered strong enough by the CDK team, we should not implement 
+this feature.
+
+### What is the technical solution (design) of this feature?
+
+Advisory information will be available as a static file, served from GitHub. 
+The CLI will consume this file from and apply a set of filters to narrow down 
+the list of advisories to the context in which it is being run. We will also 
+implement some GitHub actions to validate and copy the contents of the file over
+to the issue. For a more detailed explanation, see the Appendix.
```

Comment:
What happens in network-isolated/controlled environments? The documentation needs to be EXTREMELY clear that this is calling out to some GitHub-controlled URL. AppSec might also somewhat frown on that premise...